psse/psse-converter/src/main/java/com/powsybl/psse/converter/TransformerConverter.java

```diff
@@ -200,12 +203,15 @@ private void createThreeWindingsTransformer() {
         // move ysh between w1 and z
         TapChanger tapChanger1AdjustedYsh = tapChangerAdjustmentAfterMovingShuntAdmittanceBetweenRatioAndTransmissionImpedance(tapChanger1);
 
+        String name = psseTransformer.getName();
```

Comment:
When the name contains only white chars, it would be better to return `null` (see comments on this PR).